spec/onchain.tex

```diff
@@ -414,19 +476,39 @@ \subsection{Close Transaction}\label{sec:close-tx}
 		\[
 			\eta' = \eta
 		\]
-		\item $\mathsf{Current}$: Closing snapshot refers to current state version $v$ and any UTxO to decommit need to be present in the closed state too.
-		\[
-			\msVfy(\hydraKeys,(\cid || v || s' || \eta' || \eta_{\Delta}'),\xi) = \true
-		\]
-		which implies $\eta_{\Delta}' = \eta_{\omega}$ while $\eta_{\omega}$ does not need to be provided.
-		\item $\mathsf{Outdated}$: Closing snapshot refers the previous state $v - 1$ and any UTxO to decommit of the closing snapshot must not be recorded in the closed state.
-		\[
-			\eta_{\Delta}' = \bot
-		\]
-		\[
-			\msVfy(\hydraKeys,(\cid || v - 1 || s' || \eta' || \eta_{\omega}),\xi) = \true
-		\]
-		where $\eta_{\omega}$ is provided by the redeemer\footnote{$\eta_{\omega}$ needs to be provided to verify the signature, but is otherwise not relevant for the closed state}.
+		\red{
+			\[
+			\theta = Normal
+			\]
```

Comment:
This check is pointless: $\theta$ is not linked with anything on-chain and can be freely picked in the redeemer.